scripts/run_pytest.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
```

Comment:
I find this is helpful for my local development

Reply:
From a formal perspective, this should be a second PR. In this case, I won't block on this.

Why are you putting pytest in the script but not flake8, derp, or validate-version-bump.sh?

Is the directory modification necessary? The test dir should probably be tests not ..

My current git hook that could probably use an update:
```
if [ "`git rev-parse --abbrev-ref HEAD`" == "main" ];
     then echo "On main branch";
     exit 1;
fi                                 &&
scripts/validate-version-bump.sh   &&
flake8 src                         &&
pytest --quiet --cov=src --cov-report term-missing:skip-covered   \
  --cov-config=tox.ini --no-cov-on-fail --cov-fail-under=100 -x tests
```

Reply:
>Why are you putting pytest in the script but not flake8, derp, or validate-version-bump.sh?

Because it is quite easy to remember the command `flake8 src`, and harder to type out the pytest command with specific coverage flags.

> Is the directory modification necessary

Yes, so the script can be invoked regardless of where your terminal is currently located.

> The test dir should probably be tests not ..

Copied directly from the previous `.travis.yml` file. We can change to `tests` if desired.

>My current git hook that could probably use an update:

Probably good to update that to call this script directly after this PR 🙃

Reply:
I do see a bug in my directory modification, which I will fix now.

Reply:
Fixed in this revision.